kubeflow/core/kubeform_spawner.py

```diff
@@ -177,6 +177,26 @@ def _expand_user_properties(self, template):
 c.KubeSpawner.singleuser_working_dir = '/home/jovyan'
 volumes = []
 volume_mounts = []
+
+# Allow environment vars to override uid and gid.
+# This allows local host path mounts to be read/writable
+env_uid = os.environ.get('NOTEBOOK_UID')
+if env_uid and env_uid != 'null':
+    c.KubeSpawner.singleuser_uid = int(env_uid)
+env_gid = os.environ.get('NOTEBOOK_GID')
+if env_gid and env_gid != 'null':
```

Comment:
Won't this break by default? It looks like you are defaulting env_gid to 100. So you will now add a post pod hook that tries to mount local notebooks into the container and that command will presumably fail.

Reply:
You are correct. Updating above default values of params to null